edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/RocksDbOptionsProvider.cs

```diff
@@ -40,13 +41,21 @@ public class RocksDbOptionsProvider : IRocksDbOptionsProvider
         // Set to a limit less than a 32 bit #
         const ulong HardPendingCompactionBytes = 1024UL * 1024UL * 1024UL;
 
+        // Once write-ahead logs exceed this size, we will start forcing the flush of
+        // column families whose memtables are backed by the oldest live WAL file
+        readonly ulong maxTotalWalSize = 512 * 1024 * 1024;
+
         readonly ISystemEnvironment env;
         readonly bool optimizeForPerformance;
 
         public RocksDbOptionsProvider(ISystemEnvironment env, bool optimizeForPerformance)
         {
             this.env = Preconditions.CheckNotNull(env);
             this.optimizeForPerformance = optimizeForPerformance;
+            if (ulong.TryParse(Environment.GetEnvironmentVariable("RocksDB_MaxTotalWalSize"), out ulong envMaxTotalWalSize))
```

Comment:
Nit: For consistency, we should get the value from DependencyManager.cs and pass it down. That is how we do it everywhere else in the code.

Reply:
I cherry picked this from master. Can we merge as is and then fix master?